client/blocks/post-status/test/index.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import identity from 'lodash/identity';
+import Gridicon from 'components/gridicon';
+
+/**
+ * Internal dependencies
+ */
+import useFakeDom from 'test/helpers/use-fake-dom';
+
+describe( 'PostStatus', ( ) => {
+	useFakeDom();
```

Comment:
Seems like we don't actually need this, so this and the matching import can be reviewed.